### DISCUSSION TITLE
Deploying with Github Actions - Not working

### DISCUSSION THREAD

**Discussion by AdamFermier** [google/docsy#2036] [Q&A] [answered]
Hi Folks!  I want to deploy docsy with Github Actions and failing to succeed - so looking for either an example or help.  My resources are:

https://github.com/AdamFermier/madsciguys/blob/main/.github/workflows/static.yml
https://github.com/AdamFermier/madsciguys/actions/runs/9164732424

The github run says it was a success but the pages I can not find :)

Thanks in advance - awesome tool!  Adam

**Comment by AdamFermier** [comment, marked as answer]
Found my answer here - https://taydenwhite.github.io/docsy/about/ this has helped me deploy the site successfully.  Thanks!

**Reply by NoMeatNo** [reply]
Very interesting, @AdamFermier  Thanks for sharing!

I was looking for something similar but without any luck! I’m really curious—how did you find this web address and method? Do you remember?

**Reply by AaryaDani** [reply]
Hey, I actually work with Adam Fermier. The first challenge that he gave Tayden and I was to get this website up and running. Here is my website with a tutorial on how to deploy the website, basically running it through -savedev docsy is the method: https://aaryadani.github.io/. If you have any questions about that or the rest of my site, please let me know!
